Services/UI/OJS.Services.Ui.Business/Implementations/ContestsBusinessService.cs

```diff
@@ -68,9 +68,15 @@ public async Task<RegisterUserForContestServiceModel> RegisterUserForContest(int
 
             var contest = await this.contestsData.OneById(id);
 
-            await this.ValidateContest(contest!, user.Id!, user.IsAdmin, official);
+            var validationResult = this.contestValidationService.GetValidationResult((contest, user.Id, user.IsAdmin, official) !);
```

Comment:
space between closing bracket and ! symbol

Reply:
It is because of a stylecop rule. An error is thrown otherwise saying "Closing parenthesis should be followed by a space."